### ISSUE TITLE
Compiled Views are not in sync with Views provided in the documentation

### ISSUE THREAD

**Issue by SteveVaneeckhout** [umbraco/Umbraco.Forms.Issues#927]
Are the views that are provided in [this section](https://our.umbraco.com/documentation/Add-ons/UmbracoForms/Developer/Themes/#creating-a-theme) still up to date?

When I use them (without modifying) the conditional fields don't work. Remove the directory under themes and the conditional field work again.

Personally, I think putting the views in a zip file and linking them from the documentation is not a good idea. Keep them in the Views directory as it used to be in v7, so at least the default theme keeps working and stays up to date.

**Comment by AndyButland** [comment]
No, they aren't.  There's a pending PR to update this in the docs, and we've set up a short URL to make this easier to keep up to date with new release.  So you should always be able to get the latest from [here](https://umbra.co/umbraco-forms-default-theme).

I don't think we'll be reverting back to shipping these views on disk - that had it's own issues.  Hopefully with this short link we can ensure the docs always point to the latest, so it's not too hard to find them when building a custom theme.
